main/2021/CG-08-03.md

```diff
@@ -27,8 +27,16 @@ Installation is required, see the calendar invite.
     1. Review of action items from prior meeting.
     1. Update on [Branch Hinting](https://github.com/WebAssembly/branch-hinting) [20 min].
         1. Poll for phase 3
-    1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [30 min].
-        1. Poll for maintaining single list-of-USV `string` type
+    1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [40 min].
+        1. Poll for supporting [UTF-16](https://github.com/WebAssembly/interface-types/issues/136#issuecomment-861799460) lifting and lowering
+        1. Poll for supporting compact UTF-16 (Latin1) lifting and lowering
```

Comment:
If we're adding "compact" forms (and I don't think we should), then you might want to include `ASCII` as well.

Reply:
ASCII is both a subset of UTF-8 and Latin1, so that should be sufficiently covered from both perspectives. Wdyt?

Reply:
Given the relative lack of discussion of the "compact-utf16" idea and the fact that we could probably spend the entire 40min timeslot on just this one topic, I really don't think "compact utf16" belongs in the agenda item.

Reply:
Would UTF-16 alone be OK, though?

Reply:
> ASCII is both a subset of UTF-8 and Latin1, so that should be sufficiently covered from both perspectives. Wdyt?

Yes, it's a subset, so you can lift from ASCII to UTF-8, but you cannot lower UTF-8 to ASCII.

Reply:
What exactly are you proposing? Adding a poll on ASCII or not? What about UTF-16, would that be OK?

Reply:
Yes, I think UTF-16 is perfectly fine to poll, as discussed in https://github.com/WebAssembly/interface-types/issues/136.

As for the "compact" UTF-16, I agree with @lukewagner that we're going to waste meeting time discussing compact encodings, and getting consensus for supporting UTF-16 should already open the door for multiple encodings.

I think you should either drop it, or at least move it at the end of the agenda (I don't really have a preference here), but if you're going to keep it, then we might as well include ASCII for completness.

Reply:
Thanks, makes sense. I'll reconsider polling for only UTF-16.